webrtc.html

```diff
@@ -2389,6 +2389,58 @@
           attribute is the track that is associated with this
           <code><a>RTCRtpSender</a></code> object.</p>
         </dd>
+
+        <dt>Promise&lt;void&gt; replaceTrack (MediaStreamTrack withTrack)</dt>
```

Comment:
OK, that makes sense. Otherwise, I can imagine people thinking that |track| is the track to be replaced.